### ISSUE TITLE
Error: The class/prototype returned

### ISSUE THREAD

**Issue by oneabxv3all** [discord-buttons/discord-buttons#155]
hello sir
i have error while running script
    "discord.js": "^12.5.3",
      "discord-buttons": "^4.0.0",
Error: The class/prototype returned from the extender function must extend the existing structure class/prototype (received function ExtendedTextChannel extends TextChannel; expected extension of ExtendedTextChannel).
    at Function.extend (/home/runner/Frequency-1/node_modules/discord.js/src/util/Structures.js:82:13)
    at module.exports (/home/runner/Frequency-1/node_modules/discord-buttons/src/index.js:24:16)
    at Object.<anonymous> (/home/runner/Frequency-1/commands/main/invite.js:4:1)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at /home/runner/Frequency-1/handlers/command.js:19:24Hint:

**Comment by AngeloCore** [comment]
`The class/prototype returned from the extender function must .... (In simpler terms)`

Reason:
What this error means is that you are passing in the discord client to the package more than once. 

Fix:
Make sure you only pass in the client once to the package 
//I'm main file...
```js
require('discord-buttons')(client)
```

Or

```js
const disbut = require('discord-buttons')
disbut(client)
```

**But I have multiple clients...**
If you are using multiple instance of discord client, you can use `multipleImport` to pass in your clients
```js
//In main file
const disbut = require('discord-buttons')
disbut.multipleImport(client1, client2, ...)
```


**I have a command handler, what now?**
You can use the package by using 
`const disbut = require('discord-buttons')`
in the command file.
Note: you should have `require('discord-buttons')(client)` in your main file as well!
